includes/Core/User_Input/User_Input_User_Settings.php

```diff
@@ -0,0 +1,65 @@
+<?php
+/**
+ * Class Google\Site_Kit\Core\User_Input\User_Input_User_Settings
```

Comment:
Same as above. Does `User_Specific_Questions` make sense? Let me know!

Reply:
Updated, thank you!